client/connection.cpp

```diff
@@ -14,6 +14,10 @@
 
 namespace torchcraft {
 
+///////////////////////// LOCAL DECLARATIONS /////////////////////////////////
+
+static const std::string ERRMSG_TIMEOUT_EXCEEDED = "Timeout exceeded";
```

Comment:
Nit: A little too late but in the future, consider moving declarations like this into an anonymous namespace without `static`. Style nit: caps.